test/bdd/pkg/waci/credential_application_drivers_license.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
is it possible to move these files (and the others in the `waci` folder) to a `testdata` subdirectory like elsewhere in `test/bdd/pkg` ?

Reply:
Done!